src/CMakeLists.txt

```diff
@@ -104,14 +104,14 @@ if (HWINFO_DISK)
     add_library(hwinfo_disk_static STATIC ${DISK_SRC_FILES})
     target_include_directories(hwinfo_disk_static PUBLIC $<BUILD_INTERFACE:${HWINFO_INCLUDE_DIR}> $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
     if(APPLE)
-        target_link_libraries(hwinfo_disk_static PRIVATE "-framework IOKit" "-framework CoreFoundation")
+        target_link_libraries(hwinfo_disk_static PRIVATE "-framework IOKit" "-framework CoreFoundation_static")
```

Comment:
CoreFoundation is an OSX framework, there is no CoreFoundation_static available as far as I know...